### DISCUSSION TITLE
share links

### DISCUSSION THREAD

**Discussion by MaksGill** [outline/outline#5124] [General]
Hey, guys. Please tell me how you can view links to non-registered users in Outline, if these links were shared by registered users of Outline.

**Comment by sokai** [comment]
Hey, his may be an _old_ question, that _maybe_ is fixed by [Public sharing of collections #1160](https://github.com/outline/outline/issues/1160)(?)

**Idea:** Is it an option to show for unauthorised access on `/` similar content as on the "Shared Links" page (in the admin prefs = `/settings/shares`) and a (SSO) login link/button would be IMHO also useful …

FYI, here are some similar issues/discussions:
* #10190
    * (Repost of [#1160 (comment)](https://github.com/outline/outline/issues/1160#issuecomment-3290483349)
* #2412

Thanks and BR!
